### ISSUE TITLE
Protractor migration

### ISSUE THREAD

**Issue by ngaruko** [medic/medic-webapp#8053]
**Describe the issue**
Protractor support has stopped. So we need to migrate as soon as we can
**Describe the improvement you'd like**
See remaining test in this [spreadsheet](https://docs.google.com/spreadsheets/d/1pxOJcl1MgfK8Csh6dx0lSZRRSbgJ3KpqEBJqb9H9fsE/edit#gid=0) > 
**Describe alternatives you've considered**
Have been running protractor and wdio in parallel.

**Comment by garethbowen** [comment]
@ngaruko I think this is a duplicate of #7723
